libs/langgraph/tests/test_prebuilt.py

```diff
@@ -76,30 +68,12 @@ def test_no_modifier(checkpointer: Optional[BaseCheckpointSaver]):
     if checkpointer:
         saved = checkpointer.get_tuple(thread)
         assert saved is not None
-        assert saved.checkpoint == {
-            "v": 1,
-            "ts": AnyStr(),
-            "id": AnyStr(),
-            "channel_values": {
-                "messages": [
-                    _AnyIdHumanMessage(content="hi?"),
-                    AIMessage(content="hi?", id="0"),
-                ],
-                "agent": "agent",
-            },
-            "channel_versions": {
-                "__start__": 2,
-                "messages": 3,
-                "start:agent": 3,
-                "agent": 3,
-            },
-            "versions_seen": {
-                "__input__": {},
-                "__start__": {"__start__": 1},
-                "agent": {"start:agent": 2},
-            },
-            "pending_sends": [],
-            "current_tasks": {},
+        assert saved.checkpoint["channel_values"] == {
```

Comment:
why does this need to change?

Reply:
i removed because of the different version types/formats across different checkpointers, but i can also make something like `AnyVersion` that will just ignore those fields / customize memory saver for tests to return strings. wdyt?

Reply:
yea I think that's better